format-diff.py

```
@@ -37,6 +38,14 @@ def try_find_clang_format(exec_path):
             return c
     return candidates[-1]
 
+
+def get_clang_format_version(clang_format_cmd):
+    proc = subprocess.run([clang_format_cmd, '--version'],
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)